### ISSUE TITLE
Fix MultiAddress type for Polkadot config

### ISSUE THREAD

**Issue by jsdw** [paritytech/subxt#613]
Our Polkadot config uses `MultiAddress<AccountId32, u32>` (same as substrate), but looks like it should actually be `MultiAddress<AccountId32, ()>`. Let's double-check this and make the change.

See https://github.com/paritytech/subxt/pull/593#discussion_r938623996

**Comment by jsdw** [comment]
So for this, let's:

- Look at the above link and check that in Polkadot, we should change the MultiAddress type in our `subxt::config::PolkadotConfig` type (may mean that we'll just copy+paste it from Substrate config and change it rather than using `WithExtrinsicParams`)
- Find out what the correct version should be in the Substrate config; maybe it's `MultiAddress<AccountId32, ()>` in the default substrate node too, so we can just change it for the `subxt::config::SubstrateConfig` config and then it'll be applied to `PolkadotConfig` autoamtically anyway.

**Comment by tadeohepperle** [comment]
@jsdw Can we close this issue or do we want to investigate it further? [The PR](https://github.com/paritytech/subxt/pull/943) was merged.

**Comment by niklasad1** [comment]
Yes, I regard it as closed however we may want to open another issue regarding `SubstrateConfig` vs `SubstrateNodeTemplateConfig` and https://github.com/paritytech/subxt/pull/943#issuecomment-1531400173 explains it quite well.

**Comment by jsdw** [comment]
Agreed :)
